src/main/java/eu/europeana/metis/sandbox/service/workflow/HarvestServiceImpl.java

```diff
@@ -331,4 +332,9 @@ private RecordInfo handleDuplicated(String providerId, Step step, Record.RecordB
     saveErrorWhileHarvesting(recordToHarvest, providerId, step, new RuntimeException(recordErrorCreated.getMessage()));
     return null;
   }
+
+  private String createTemporaryIdFromPath(String extractedDirectory, Path pathToRelativize){
+    return !extractedDirectory.isBlank() ? Paths.get(extractedDirectory).relativize(pathToRelativize).toString() :
+            pathToRelativize.getFileName().toString();
+  }
 }
```

Comment:
I ran it locally and is asking for the `TestUtils` 
.../metis-sandbox/src/test/java/eu/europeana/metis/sandbox/common/TestUtils.java
to implement this methods the `TestHttpRecordIterator` class.
```
    @Override
    public String getExtractedDirectory() {
      return this.extractedDirectory != null ? this.extractedDirectory.toString() : "";
    }
```

Reply:
Oops! I don't know how I missed that, I'll fix it